src/inference/dev_api/openvino/runtime/threading/istreams_executor.hpp

```diff
@@ -150,6 +150,12 @@ class OPENVINO_RUNTIME_API IStreamsExecutor : virtual public ITaskExecutor {
               _threadPreferredCoreType(threadPreferredCoreType),
               _streams_info_table{streamsInfoTable},
               _cpu_reservation{cpuReservation} {}
+
+        static Config update_executor_config(const Config& initial,
```

Comment:
Why is it static method? It makes a copy of the config, and I don't really see any reason to do it.

Reply:
Static function is not necessary, it has been changed to non-static function.